SingularityUI/app/components/common/modal/FormModal.jsx

```diff
@@ -2,10 +2,27 @@ import React from 'react';
 import classNames from 'classnames';
 
 import { Modal, Button } from 'react-bootstrap';
-import TagsInput from 'react-tagsinput'
-import Duration from './formItems/Duration';
+import TagsInput from 'react-tagsinput';
+import Duration from '../formItems/Duration';
 
 export default class FormModal extends React.Component {
+  constructor(props) {
+    super(props);
+    const formState = {};
+    _.each(props.formElements, (e) => {
+      formState[e.name] = e.defaultValue && e.defaultValue.toString();
+    });
```

Comment:
I think the reason this got highlighted is a indentation thing, but I can change that. I prefer the ES6 one, but I think in some of the other PRs we've had usage of underscore's one.